sql.go

```diff
@@ -179,155 +182,156 @@ const IS = 57422
-const PRIMARY = 57451
-const COLUMN = 57452
-const CONSTRAINT = 57453
-const SPATIAL = 57454
-const FULLTEXT = 57455
-const FOREIGN = 57456
-const KEY_BLOCK_SIZE = 57457
-const SHOW = 57458
-const DESCRIBE = 57459
-const EXPLAIN = 57460
-const DATE = 57461
-const ESCAPE = 57462
-const REPAIR = 57463
-const OPTIMIZE = 57464
-const TRUNCATE = 57465
-const MAXVALUE = 57466
-const PARTITION = 57467
-const REORGANIZE = 57468
-const LESS = 57469
-const THAN = 57470
-const PROCEDURE = 57471
-const TRIGGER = 57472
-const VINDEX = 57473
-const VINDEXES = 57474
-const STATUS = 57475
-const VARIABLES = 57476
-const BEGIN = 57477
-const START = 57478
-const TRANSACTION = 57479
-const COMMIT = 57480
-const ROLLBACK = 57481
-const BIT = 57482
-const TINYINT = 57483
-const SMALLINT = 57484
-const MEDIUMINT = 57485
-const INT = 57486
-const INTEGER = 57487
-const BIGINT = 57488
-const INTNUM = 57489
-const REAL = 57490
-const DOUBLE = 57491
-const FLOAT_TYPE = 57492
-const DECIMAL = 57493
-const NUMERIC = 57494
-const TIME = 57495
-const TIMESTAMP = 57496
-const DATETIME = 57497
-const YEAR = 57498
-const CHAR = 57499
-const VARCHAR = 57500
-const BOOL = 57501
-const CHARACTER = 57502
-const VARBINARY = 57503
-const NCHAR = 57504
-const TEXT = 57505
-const TINYTEXT = 57506
-const MEDIUMTEXT = 57507
-const LONGTEXT = 57508
-const BLOB = 57509
-const TINYBLOB = 57510
-const MEDIUMBLOB = 57511
-const LONGBLOB = 57512
-const JSON = 57513
-const ENUM = 57514
-const GEOMETRY = 57515
-const POINT = 57516
-const LINESTRING = 57517
-const POLYGON = 57518
-const GEOMETRYCOLLECTION = 57519
-const MULTIPOINT = 57520
-const MULTILINESTRING = 57521
-const MULTIPOLYGON = 57522
-const NULLX = 57523
-const AUTO_INCREMENT = 57524
-const APPROXNUM = 57525
-const SIGNED = 57526
-const UNSIGNED = 57527
-const ZEROFILL = 57528
-const DATABASES = 57529
-const TABLES = 57530
-const VITESS_KEYSPACES = 57531
-const VITESS_SHARDS = 57532
-const VITESS_TABLETS = 57533
-const VSCHEMA_TABLES = 57534
-const EXTENDED = 57535
-const FULL = 57536
-const PROCESSLIST = 57537
-const NAMES = 57538
-const CHARSET = 57539
-const GLOBAL = 57540
-const SESSION = 57541
-const ISOLATION = 57542
-const LEVEL = 57543
-const READ = 57544
-const WRITE = 57545
-const ONLY = 57546
-const REPEATABLE = 57547
-const COMMITTED = 57548
-const UNCOMMITTED = 57549
-const SERIALIZABLE = 57550
-const CURRENT_TIMESTAMP = 57551
-const DATABASE = 57552
-const CURRENT_DATE = 57553
-const CURRENT_TIME = 57554
-const LOCALTIME = 57555
-const LOCALTIMESTAMP = 57556
-const UTC_DATE = 57557
-const UTC_TIME = 57558
-const UTC_TIMESTAMP = 57559
-const REPLACE = 57560
-const CONVERT = 57561
-const CAST = 57562
-const SUBSTR = 57563
-const SUBSTRING = 57564
-const GROUP_CONCAT = 57565
-const SEPARATOR = 57566
-const MATCH = 57567
-const AGAINST = 57568
-const BOOLEAN = 57569
-const LANGUAGE = 57570
-const WITH = 57571
-const QUERY = 57572
-const EXPANSION = 57573
-const UNUSED = 57574
+const STARTS_WITH = 57426
```

Comment:
If we got that far, should we just remove everything we don't support? Also the reason when we have `ExecutionStatus` but not just `Status` is because `STATUS` is reserved keyword. If you remove it, we can make it at least an alias to `ExecutionStatus` (column name in UI says "Status")

Reply:
That's the plan. I'll start cleaning up soon.